rtf.js

```diff
@@ -1931,8 +1959,17 @@ RTFJS.Document.prototype.parse = function(blob, renderer) {
 				if (isNaN(param))
 					throw new RTFJS.Error("Could not parse hexadecimal number");
 				
-				if (process != null)
-					appendText(cptable[parser.codepage].dec[param]);
+				if (process != null) {
+					// Looking for current fonttbl charset
+					var codepage = parser.codepage;
+					if (parser.state.chp.hasOwnProperty("fontfamily")) {
+						idx = parser.state.chp.fontfamily;
+						if (inst._fonts != undefined && inst._fonts[idx] != null)
```

Comment:
Please add `&& inst._fonts[idx].charset != undefined` to verify there is a charset defined for this font as the property is optional and we might not have a mapping (in case of codepage 42).